### ISSUE TITLE
The "mysterious" app_list breadcrumb

### ISSUE THREAD

**Issue by skndn60** [darklow/django-suit#74]
Not sure whether this is an issue DS must/can solve but I thought it's worth reporting.

When using DS one has the option to directly open a `change_list` on the first model of the app. The breadcrumbs show up correctly, eg:

```
Home >> model_list
```

A great feature as it hides the utterly silly and useless standard Django app_list level. However, when one next chooses an instance from the `change_list`, app_list level is added to the breadcrumbs in `change_form`:

```
Home >> app_list >> model_list >> model
```

Basically a useless and confusing addition with its origins in plain vanilla Django.

The app_list crumb doesn't point to a any meaningful url, so it takes the user to the admin home. A confusing thing...

The solution to this would be to modify the change_form templates and eliminate that level from `{% block breadcrumbs %}` but maybe there's a nicer solution?
